### DISCUSSION TITLE
Arrange fields in rows rather than only vertically stacked

### DISCUSSION THREAD

**Discussion by angrybrad** [craftcms/cms#8107] [Ideas]
> Created by: John Baxter (john@luckyturnmedia.co.uk) on 2015/03/11 09:07:25 +0000
> Votes at time of UserVoice import: 68

-----------------------------------

It would be good to be able to arrange fields in rows when adding them to tabs. If i have a bunch of fields that are switches/controls then having them stacked vertically doesn't make sense visually.

**Comment by angrybrad** [comment]
> Posted by Christian (seelbach@gmx.de) on 2015/03/09 00:49:13 +0000
-----------------------------------

Julian, SuperTable is not interchangable with this requested solution. Having the fields directly accessible form the entry model allows to set field parameters on the criteria model, e.g. `craft.entries.myLightSwitch(1).myDateTime(&#39;&gt; 2015-08-27&#39;)`. This is not possible with fields inside a Matrix and/or SuperTable.

**Comment by angrybrad** [comment]
> Posted by Julian Rutherford (justjoolz@gmail.com) on 2015/04/19 11:42:02 +0000
-----------------------------------

Take a look at this, I wish I&#39;d seen this the first day I started developing with craft !
https://github.com/engram-design/SuperTable

**Comment by angrybrad** [comment]
> Posted by Christian (seelbach@gmx.de) on 2015/03/09 00:49:13 +0000
-----------------------------------

Yes. This suggestion, plus being able to change the fields&#39; CP name and label, maybe the ability to add and edit notes/instructions directly from the field layout designer would be so awesome!

**Comment by brandonkelly** [comment]
Related to #1103 and #1138.
